frontend/src/components/BlogsList.tsx

```diff
@@ -33,7 +33,7 @@ const BlogsList = () => {
       const timer = setTimeout(() => {
       setShowEndMessage(true);
       setShowConfetti(true);
-      }, 2000);
+      }, 4000);
```

Comment:
What's the reasoning behind this change ?

Reply:
When a user scrolls quickly on the landing page, the lazy loading takes a few milliseconds to start. During this brief period, the page isn't loading, and the end message appears too soon. To fix this, I added a timer to give the lazy loading enough time to run. Initially, I set the timer to 2 seconds, which worked fine on my local machine. However, maybe due to network delays on the online hosted landing page, it takes a bit longer for lazy loading to start. So, I increased the timer to ensure it works correctly.

Reply:
and also here by quickly i mean very rapidly, but if you scroll smoothly you will not see this bug.